aten/src/ATen/Context.h

```diff
@@ -320,6 +320,8 @@ class TORCH_API Context {
   void setFloat32MatmulPrecision(const std::string& s);
   bool allowTF32CuDNN() const;
   void setAllowTF32CuDNN(bool);
+  bool allowTF32oneDNN() const;
+  void setAllowTF32oneDNN(bool);
```

Comment:
It should be `OneDNN` for code style consistency.

Reply:
Thank for suggestions! I have changed the name to `OneDNN` in PR,  I've already synced the change with Yanbing.